ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.html

```diff
@@ -31,13 +31,13 @@
         {{sample.sampleStatus}}
       </td>
       <td>
-        <ng-container *ngIf="sample.sampleType === 'Normal'">
+        <ng-container *ngIf="sample.sampleType === 'Normal' && canChangeCollectionDate()">
           <app-field-datepicker (dateChanged)="valueChanged($event, 'collectionDate', sample)" [allowUnknownDay]="false"
                                 [colorDuringPatch]="isPatchedCurrently('collectionDate')"
                                 [dateString]="sample.collectionDate"
                                 [showTodayButton]="true"></app-field-datepicker>
         </ng-container>
-        <ng-container *ngIf="sample.sampleType === 'Tumor'">
+        <ng-container *ngIf="sample.sampleType === 'Tumor' || !canChangeCollectionDate()">
```

Comment:
I also removed the date submission, but we are waiting on the final decision on that